.github/workflows/main.yml

```diff
@@ -0,0 +1,198 @@
+name: Main
```

Comment:
the create release and kodi build also seem to be missing. We need to add them back.

Reply:
How is your release workflow ?

Reply:
The release for the major version is edited with latest details:

https://github.com/IgnisDa/ryot/blob/15a3fe9c35675184e662ea774f6feb71415cbbe8/.github/workflows/release.yml#L26-L63

Kodi artifacts are built and deployed, docs are updated and the demo instance is also redeployed:

https://github.com/IgnisDa/ryot/blob/15a3fe9c35675184e662ea774f6feb71415cbbe8/.github/workflows/release.yml#L137-L195

Reply:
I think this flow might need to be revisited. Does not seem optimal and I have had a few users complain about it not being very predicatable.

Reply:
One thing I will add is the workflow should not depend on the `create-release` action (at least not the PR workflow) because if the contributors want to test on their local repo, they have to create a release.

Reply:
Yeah I'll make the workflow create a new release with auto generated body for only tagged released and attach the kodi archive as an artifact.

Reply:
Do you need my help on anything else ?

Reply:
What's the use of building the frontend twice? I think we can get rid of the `build-frontend` step completely and do that part inside the dockerfile.

Reply:
Made the changes, can you take a look and see if it looks fine?

Reply:
Yeah if it work, it's fine. That is because I dont know how moon works so I just copy from the old dockerfile but I am happy that we dont need to build the frontend twice